dolphinscheduler-task-plugin/dolphinscheduler-task-python/src/main/java/org/apache/dolphinscheduler/plugin/task/python/PythonTask.java

```diff
@@ -95,8 +100,8 @@ public void setCommand(String command) {
     public void handle() throws Exception {
         try {
             //    construct process
+            this.command = buildCommand();
             TaskResponse taskResponse = pythonCommandExecutor.run(command);
```

Comment:
Good job.
It seems we need deelete org.apache.dolphinscheduler.spi.task.AbstractTask#setCommand, it is better to use local variables.

Reply:
Did you mean it is better to use local variables like `pyCommand` but NOT `this.command`?

Reply:
```
String command=buildCommand(); 
```
you can refer to the ShellTask.

Reply:
got it~

Reply:
The code has been updated

Reply:
also need to delete AbstractTask#setCommand

Reply:
AbstractTask#setCommand has been deleted